composer.json

```diff
@@ -35,7 +35,11 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "scripts": {
-      "phing": "./vendor/bin/phing test",
+      "pre-commit": [
```

Comment:
The idea is that we run this as `composer pre-commit` manually, or does it always run for every commit you do?
I'd prefer it not to run automatically for every commit because sometimes commits are just temporarily broken by design (e.g. TDD, or when refactoring but splitting up the work in multiple commits)

Reply:
And if we run it manually it might be nice to have something shorter like `composer ci`?

Reply:
I couldn't really find a good name but I like ci. It's short and covers the load.